### PR TITLE
GRF-308: Display text & textarea attribute values

### PR DESCRIPTION
**Pull request by BryanS29-akeneo**
**Description (for Contributor and Core Developer)**

Display text & textareas attribute values already done in [GRF-109](https://github.com/akeneo/pim-community-dev/pull/17868).
This PR is just about renaming `identifier` by `uuid` in front size because this is what it is.

**Definition Of Done (for Core Developer only)**

- [ ] Tests
- [ ] Migration & Installer
- [ ] PM Validation (Story)
- [ ] Changelog (maintenance bug fixes)
- [ ] Tech Doc

### CODE REVIEW

**Review by FSAkeneo — APPROVED** [review]

File: src/Akeneo/Category/front/src/feature/models/Attribute.ts [hunk 1]
```diff
@@ -1,5 +1,6 @@
 import {LabelCollection} from '@akeneo-pim-community/shared';
 
+// TODO keep or remove ?
```
Comment (antoinetrouve) [inline src/Akeneo/Category/front/src/feature/models/Attribute.ts:3]: usefull ?

File: src/Akeneo/Category/back/Domain/Model/Attribute/Attribute.php [hunk 2]
```diff
@@ -44,7 +44,7 @@ public function __construct(
     public function normalize(): array
     {
         return [
-            'identifier' => (string) $this->uuid,
+            'uuid' => (string) $this->uuid,
```
Comment (antoinetrouve) [inline src/Akeneo/Category/back/Domain/Model/Attribute/Attribute.php:47]: All test are ok ?

**Review by antoinetrouve — APPROVED** [review]

File: src/Akeneo/Category/back/Domain/Model/Attribute/Attribute.php (hunk 2 above)
Comment (amigaire) [reply src/Akeneo/Category/back/Domain/Model/Attribute/Attribute.php:47]: We call it uuid, but we keep 'template_identifier', not 'template_uuid' ?

Comment (BryanS29-akeneo) [reply src/Akeneo/Category/back/Domain/Model/Attribute/Attribute.php:47]: yes you are right @amigaire, done

**Pull request merged**